README.md

```diff
@@ -88,179 +77,9 @@ To perform these tests, you'll need some test Bitcoins on the Bitcoin testnet, a
-
-    **Note**: again, wait until you see at least 30 confirmations before proceeding to the next step.
-
-3. Verify the Ethereum transaction
-    ```
-    axelarcli tx ethereum verify-erc20-deposit {txID} {amount} {deposit addr} --from validator -y -b block
-    -> wait for verification to be confirmed (~10 Axelar blocks)
-    ```
-    Here, amount should be specific in Satoshi. (For instance, 0.0001BTC = 10000 Satoshi)
-
-    e.g.,
-    ```
-    axelarcli tx ethereum verify-erc20-deposit 0x01b00d7ed8f66d558e749daf377ca30ed45f747bbf64f2fd268a6d1ea84f916a 10000  0x5CFEcE3b659e657E02e31d864ef0adE028a42a8E --from validator -y -b block
-    -> wait for verification to be confirmed (~10 Axelar blocks, 1 minute)
-    ```
 
-2. Trigger signing of all pending transfers to Bitcoin
-    ```
-    axelarcli tx bitcoin sign-pending-transfers {tx fee} --gas="auto" --gas-adjustment=1.15 --from validator -b block -y
-    -> wait for sign protocol to complete (~10 Axelar blocks, 1 minute)
+4. Check that you received the funds
     ```
-    e.g.,
-    ```
-    axelarcli tx bitcoin sign-pending-transfers 0.0001btc --gas="auto" --gas-adjustment=1.15 --from validator -b block -y
-    ```
-
-    **Note**: If you are having gas error issue `out of gas: out of gas in location: WriteFlat; gasWanted: 200000, gasUsed: 200780"`, try to increment the `gas-adjustement` to a higher value `1.2` or `1.3`. With more deposits, the transaction become larger and hence requires more gas
-
-3. Submit the transfer to Bitcoin
-    ```
-    axelarcli q bitcoin send
-    -> returns tx ID
-    ```
-
-    You can monitor the status of your transfer using the bitcoin testnet explorer: https://blockstream.info/testnet/
-
-#### 🛑 **IMPORTANT: Verify outpoints of previous withdrawal tx (repeat for each outpoint)**
-    
-Every Bitcoin transaction can have multiple outputs. Similarly, on your withdrawal transactions you will see multiple outputs even if you processed one withdrawal (the other outputs are used for internal controller functions). Please verify all outputs (i.e., {transaction hash, index} pairs) in your withdrawal transaction.
-
-Without this step, other users of the testnet will **not** be able to withdraw their wrapped tokens. Be a good citizen and verify the outpoints!
-
-1. Create verification json object for Axelar
-    ```
-    axelarcli q bitcoin txInfo {blockhash} {txID}:{voutIdx}
-    -> returns json of verification info for the given outpoint (copy the escaped string)
-    ```
-    e.g.,
-    ```
-    axelarcli q bitcoin txInfo 4ac9dc50dc1b952cb1efca1e634216da2f5e3a12b4a4a802ce0f6b1271876bd2 da5b2e8037ce4b95f40ada01c6c2cd3ccb806d0a952906130eb9b806f7887590:1
+    axelarcli q account {validator_addr}
     ```
 
```

Comment:
Should we add one more section here that lists the exercises with links to the corresponding readme?

Reply:
we'll give the links from our discourse landing page for the exercises.